bindings/python/iota_sdk/types/payload.py

```diff
@@ -45,7 +70,19 @@ def as_dict(self):
 
 @dataclass
 class MilestonePayload(Payload):
-    """Initialize a MilestonePayload
+    """Represents a MilestonePayload. Inherits `Payload`.
+
+    Attributes:
+        index (int): corresponding milestone index
+        timestamp (int): corresponding milestone timestamp
+        protocolVersion (int): current protocol version
+        previousMilestoneId (HexStr): previous milestone id
+        parents (List[HexStr]): parents of the milestone 
+        inclusionMerkleRoot (HexStr): the inclusion merkle root
+        appliedMerkleRoot (HexStr): the applied merkle root
+        signatures (List[Ed25519Signature]): milestone signatures
+        options (List[Any], optional): milestone options
+        metadata (HexStr, optional): milestone metadata
```

Comment:
types

Reply:
More below, will not comment there now

Reply:
yeah ... that went into `part 2` (see previous commit message :stuck_out_tongue_closed_eyes: )